client/transactions/list/test/index.js

```diff
@@ -89,20 +88,26 @@ const getMockTransactions = () => [
 describe( 'Transactions list', () => {
 	beforeEach( () => {
 		jest.clearAllMocks();
+
 		// the query string is preserved across tests, so we need to reset it
-		if ( ! isEmpty( getQuery() ) ) {
-			updateQueryString( {}, '/', {} );
-		}
+		// eslint-disable-next-line camelcase
+		updateQueryString( { currency_is: 'usd' }, '/', {} );
```

Comment:
This change helped me to discover a bug with the filter:

1. Do we need to show filter at all when only one currency is supported? I'd expect no.
2. When one currency is supported and no currency_is param set, there should be a transactions summary at the bottom of the table.
3. Sad thing, but we can't rely on the account supported currencies for this filter. Here is the scenario:
  - Add second account (let's say USD while first account is EUR).
  - Make transaction with its currency (USD).
  - Make transaction with first account currency (EUR).
  - Remove second account (USD) from settings (it's possible).
  - Refresh account cache.
  - Result: no USD in filter values.